package.json

```diff
@@ -170,7 +170,6 @@
 		"env:cli": "node ./tools/local-env/scripts/docker.js run cli",
 		"env:logs": "node ./tools/local-env/scripts/docker.js logs",
 		"env:pull": "node ./tools/local-env/scripts/docker.js pull",
-		"test:php": "node ./tools/local-env/scripts/docker.js run --rm phpunit phpunit",
```

Comment:
Is there any harm in leaving this command? If some contributors prefer to run the unit tests within the Docker containers, it will still work.

Reply:
If it's preferred to remove this, then I say just rename `test:php-composer` to `test:php`.

Reply:
No harm leaving it at all AFAICS. This was the "radically clean-up" way of doing it which doesn't leave any stray (unused) commands lying about.

Reply:
I've just brought back the command now.